scripts/PRODUCTION_ENV_FIX.md

```
@@ -66,6 +66,24 @@ docker exec pulseplate-production_app_1 python -c "import os; print('APP_ENV:',
+  "version": "1.0.0",
+  "git_sha": "f4c8b72e593f",  # pragma: allowlist secret
+  "timestamp": "2026-01-01T13:25:47.488673+00:00",
+  "environment": "production"
+}
+```
+
+> Примечание:
+> Значение `git_sha` берётся из env `GIT_SHA` и приводится к короткому виду
+> (12 символов) для удобства визуальной проверки деплоя.
+> Если `GIT_SHA` отсутствует или некорректен, сервис вернёт `git_sha` как `"unknown"`.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)